.github/workflows/prepare-release.yml

```
@@ -0,0 +1,94 @@
+name: Update version on new release branch
+
+on:
+  create:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update_version:
+    if: github.ref_type == 'branch' && startsWith(github.ref, 'refs/heads/release-v')
+    runs-on: ubuntu-latest
```

Sets runner to ubuntu-24.04 to avoid unexpected breaking changes in the future
```suggestion
    runs-on: ubuntu-24.04
```